joplin/js/editor.js

```diff
@@ -48,40 +30,27 @@ $(function() {
     janisPreviewUrl: getPreviewUrl('en'),
   };
 
```

Comment:
in general the style guide anchor links don't work so well, they are out of date. but I cleaned it up somewhat, removed the truly out of date content

Reply:
Yes there is an issue in the backlog somewhere to make them work better.